### PR TITLE
Reenable IISExpress on helix

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

**Review by natemcmaster — APPROVED** [review]

File: Directory.Build.targets [hunk 1]
```diff
@@ -75,90 +75,5 @@
   <Import Project="eng\targets\Wix.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.wixproj'" />
   <Import Project="eng\targets\Npm.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.npmproj'" />
 
-  <!-- Prepares the test projects for helix by including xunit and publishing -->
```
Comment (natemcmaster) [inline Directory.Build.targets]: +1 for moving all this helix stuff to another file. I was going to do this anyways, so thanks for taking care of it.

**Comment by pakrym** [comment]
I refactored how helix works items are collected to give projects more control over queues they run on.
I've also added support for per project pre/post commands

**Review by HaoK — APPROVED** [review]
Nice!

**Comment by HaoK** [comment]
Can you also add a new section in the helix docs with some examples of how to use the new stuff?

**Pull request merged**